### PR TITLE
improving code coverage issue #615

### PR DESCRIPTION
**Pull request by Manan-S0ni**
#615
improving code coverage for zingg/common/core:
/hash to 97% coverage
model and sink both to 100% coverage

### CODE REVIEW

File: common/core/src/test/java/zingg/hash/TestIdentityLong.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class TestIdentityLong {
+
+    @Test
+    public void testIdentityLong() {
+        IdentityLong value = getInstance();
+        assertEquals(12345L, value.call(12345L));
+    }
+
+    @Test
+    public void testIdentityLong1() {
```
Comment (sonalgoyal) [inline common/core/src/test/java/zingg/hash/TestIdentityLong.java:18]: rename to testNullValue

File: common/core/src/test/java/zingg/hash/TestLessThanZeroFloat.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package zingg.hash;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
```
Comment (sonalgoyal) [inline common/core/src/test/java/zingg/hash/TestLessThanZeroFloat.java:3]: this is a good test

File: common/core/src/test/java/zingg/hash/TestTrimLastDigitsFloat.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package zingg.hash;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```
Comment (sonalgoyal) [inline common/core/src/test/java/zingg/hash/TestTrimLastDigitsFloat.java:3]: can you please test for negative values as well

File: common/core/src/test/java/zingg/hash/TestRangeBetween100And1000Long.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+package zingg.hash;
+
```
Comment (sonalgoyal) [inline common/core/src/test/java/zingg/hash/TestRangeBetween100And1000Long.java:2]: what happens if you give a range say -10 and -100? What happens with 0,0? see if you can break these functions.

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: common/core/src/test/java/zingg/common/core/sink/TestTableOutput.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
+package zingg.common.core.sink;
```
[the remark this answers is not in the document]
Comment (Manan-S0ni) [reply common/core/src/test/java/zingg/common/core/sink/TestTableOutput.java:1]: Noted

**Comment by sonalgoyal** [comment]
@vikasgupta78 can you pease get this to completion?

**Comment by vikasgupta78** [comment]
@gnanaprakash-ravi if you have time can you look at this and incorporate review comments as suggested by Sonal

**Comment by gnanaprakash-ravi** [comment]
Sure, I will look into it!!

**Comment by sonalgoyal** [comment]
I have some ideas on this one, so please talk to me before you start

**Pull request closed without merging**